zulip_bots/zulip_bots/test_lib.py

```diff
@@ -159,6 +159,13 @@ def read_bot_fixture_data(bot_name, test_name):
     http_data = json.loads(content)
     return http_data
 
+def patch_response(http_response, http_headers):
```

Comment:
I believe we should rename this function to `get_response` or `get_response_object`. Patching can be confused with [this](https://stackoverflow.com/questions/5626193/what-is-monkey-patching).